docs/sources/tempo/api_docs/_index.md

```diff
@@ -546,3 +548,63 @@ GET /status/usage-stats
 ```
 
 Displays anonymous usage stats data that is reported back to Grafana Labs.
+
+# Tempo GRPC API
```

Comment:
Making this a subheading since most pages only have one level 1 heading, which is also the page title. 
```suggestion
## Tempo GRPC API
```